main.py

```diff
@@ -1,10 +1,49 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    # Open and load JSON file to "players" variable
+    with open("players.json") as players_file:
+        players = json.load(players_file)
+        for player in players:
+            race = players[player]["race"]
+            skills = players[player]["race"]["skills"]
+            guild = players[player]["guild"]
```

Comment:
Use .items() or .values() for dict. Getting in the loop by key the same dict like: `players[player]` is so bad